change/@azure-msal-browser-9e850e9e-4ee3-4888-a5e9-6d74c343b951.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "upn test fix",
```

Comment:
Can you update these comments to something like "Update ADAL to MSAL SSO logic to use preferred_username instead of upn by default #3945"